src/clients/Deluge.ts

```diff
@@ -171,25 +188,31 @@ export default class Deluge implements TorrentClient {
 					return InjectionResult.TORRENT_NOT_COMPLETE;
 				}
 			}
-
 			const params = this.formatData(
 				`${newTorrent.name}.cross-seed.torrent`,
 				newTorrent.encode().toString("base64"),
-				path ? path : torrentInfo.save_path,
+				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+				path ? path : torrentInfo!.save_path,
```

Comment:
what is the point of this change?

Reply:
`torrentInfo` is technically undefined at that point (only defined once earlier in an if condition); this forces the acknowledgement that torrentInfo will not be undefined here.

The real fix would be a bit longer than that as it requires to rejiggle the logic

Reply:
Rejiggled a bit.